docs/snippets/ov_denormals.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <openvino/runtime/core.hpp>
+
+int main() {
+    try {
+        std::string modelPath = "modelWithDenormals.xml";
+        std::string device = "CPU";
+        ov::AnyMap config;
+        //! [ov:denormals_optimization:part0]
+        ov::Core core;                                              // Step 1: create ov::Core object
+        core.set_property(ov::denormals_optimization("YES"));       // Step 1b: Enable denormals optimization
```

Comment:
`"YES"` must be replaced with `true`, since `"YES"` is API 1.0 specific

Reply:
do you mean we need add new config_value such as `DECLARE_CONFIG_VALUE(TRUE)` and `DECLARE_CONFIG_VALUE(FALSE)` in [ie_plugin_config.hpp](ie_plugin_config.hpp), it was not decleared before, otherwise, setting to 'TRUE' can not be recognized in config read process.

Reply:
no, let's just use `core.set_property(ov::denormals_optimization(true));`

Reply:
ok, I see. done! thanks.